pkg/virt-operator/resource/generate/rbac/apiserver.go

```diff
@@ -218,6 +218,17 @@ func newApiServerClusterRole() *rbacv1.ClusterRole {
 					"get", "list", "watch",
 				},
 			},
+			{
+				APIGroups: []string{
+					"",
+				},
+				Resources: []string{
+					"namespaces",
+				},
+				Verbs: []string{
+					"get",
```

Comment:
this will have to be updated when using informers

Reply:
yup, thx